tests/demonstrator_scripts/test_computation_volume_content.py

```diff
@@ -0,0 +1,138 @@
+from pint.testsuite.helpers import assert_quantity_almost_equal as assert_approx
+from lebedigital.demonstrator_scripts.computation_volume_content import computation_volume_content
+from lebedigital.demonstrator_scripts.computation_volume_content import computation_ratios
+from lebedigital.unit_registry import ureg
+
+def test_computation_volume_content() :
+    """
+    This is the test for the computation_volume_content function.
+    It test all output with  and performs a number of sanity checks
+    """
+
+    # TEST No 1, check intended use in general
+    input1 = {}
+    # densities
+    input1['density_cem'] = 1.44 * ureg('g/cm^3')
+    input1['density_sub'] = 840 * ureg('kg/m^3')
+    input1['density_water'] = 977 * ureg('kg/m^3')
+    input1['density_plasticizer'] = 0.98 * ureg('g/cm^3')
+    input1['density_aggregates'] = 1.5 * ureg('t/m^3')
```

Comment:
should we take some realistic values? The density of aggregates is usually much larger

Reply:
I think it is off by a factor of 2, so nothing serious. Really depends on the aggregates.
Note, the unit is given in t/m^3 (because we can :D)